.github/workflows/ci.yml

```diff
@@ -35,6 +35,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Had to add this here and in two more workflows, checkout step now takes 2s instead of 1s, so we should still be fine.